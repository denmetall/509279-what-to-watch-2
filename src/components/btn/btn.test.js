import React from 'react';
import Btn from './btn.jsx';
import renderer from 'react-test-renderer';

it(`Components Btn renders correctly`, () => {
  const tree = renderer
    .create(<Btn/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
