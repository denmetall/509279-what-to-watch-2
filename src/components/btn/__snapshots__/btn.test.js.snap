// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Btn renders correctly 1`] = `
<a
  className="btn movie-card__button"
  href="add-review.html"
>
  Add review
</a>
`;
